src/app/components/tables/subject-table/subject-table.component.scss

```diff
@@ -43,5 +43,21 @@ md-cell {
 md-paginator {
   @include font-body();
 
+  background-color: transparent;
+}
+
+.bottom-bar {
+  width: 100%;
   background-color: $cl-base-20;
 }
+
+.patients {
+  float: left;
```

Comment:
avoid the use of `float` prefer `flex` instead, e.g., add a spacer in between with `flex: 1`.